pkg/controller/volume/persistentvolume/provision_test.go

```diff
@@ -210,11 +209,11 @@ func TestProvisionSync(t *testing.T) {
 			// Binding will be completed in the next syncClaim
 			newClaimArray("claim11-8", "uid11-8", "1Gi", "", v1.ClaimPending, &classGold, annStorageProvisioner),
 			[]string{"Normal ProvisioningSucceeded"},
-			[]reactorError{
+			[]pvtesting.ReactorError{
 				// Inject error to the first
 				// kubeclient.PersistentVolumes.Create() call. All other calls
 				// will succeed.
-				{"create", "persistentvolumes", errors.New("Mock creation error")},
+				{Verb: "create", Resource: "persistentvolumes", Error: errors.New("Mock creation error")},
```

Comment:
These key fields are required if we composite a public struct from another package, otherwise `composite literal uses unkeyed fields` is emitted by `go vet`.